Activities/FTP/UiPath.FTP.Activities/NetCore/ViewModels/WithFtpSessionViewModel.cs

```diff
@@ -135,12 +135,12 @@ protected override void InitializeModel()
             MenuActionsBuilder<PasswordInputMode>.WithValueProperty(PasswordInputModeSwitch)
               .AddMenuProperty(Password, PasswordInputMode.Password)
               .AddMenuProperty(SecurePassword, PasswordInputMode.SecurePassword)
-              .BuildAndInsertMenuActions();
+              .BuildAndInsertMenuActions(true);
 
             MenuActionsBuilder<PasswordInputMode>.WithValueProperty(CertificatePasswordInputModeSwitch)
               .AddMenuProperty(ClientCertificatePassword, PasswordInputMode.Password)
               .AddMenuProperty(ClientCertificateSecurePassword, PasswordInputMode.SecurePassword)
-              .BuildAndInsertMenuActions();
+              .BuildAndInsertMenuActions(false);
```

Comment:
I think that this one, although not mentioned explicitly in the ticket, suffers from the same lack of visibility and should be promoted to a main option as well. We should ask Maciej.

Reply:
I reached to Maciej and he confirmed that the changes should be made also for ClientCertificate
I will update this PR